packages/web-api-scan-job-manager/src/worker/worker.ts

```diff
@@ -51,16 +52,29 @@ export class Worker {
 
             this.poolLoadGenerator.setLastTasksIncrementCount(tasksQueuedCount);
 
+            const runningTasks = poolMetricsInfo.load.runningTasks;
+            const samplingIntervalInSeconds = this.poolLoadGenerator.samplingIntervalInSeconds;
+            const maxParallelTasks = poolMetricsInfo.maxTasksPerPool;
+
             this.logger.logInfo('Pool load statistics', {
                 activeTasks: poolMetricsInfo.load.activeTasks.toString(),
-                runningTasks: poolMetricsInfo.load.runningTasks.toString(),
+                runningTasks: runningTasks.toString(),
                 requestedTasksToAddPerInterval: tasksIncrementCount.toString(),
                 tasksAddedPerInterval: tasksQueuedCount.toString(),
-                samplingIntervalInSeconds: this.poolLoadGenerator.samplingIntervalInSeconds.toString(),
+                samplingIntervalInSeconds: samplingIntervalInSeconds.toString(),
                 processingSpeedTasksPerMinute: this.poolLoadGenerator.processingSpeedPerMinute.toString(),
                 activeToRunningTasksRatio: this.poolLoadGenerator.activeToRunningTasksRatio.toString(),
             });
 
+            const batchPoolMeasurements: BatchPoolMeasurements = {
+                runningTasks,
+                samplingIntervalInSeconds,
+                maxParallelTasks,
+            };
+
+            // tslint:disable-next-line: no-null-keyword
+            this.logger.trackEvent('ScanTasksQueued', null, batchPoolMeasurements);
```

Comment:
can we rename event to BatchPoolStats or similar?

Reply:
updated.